client/src/components/PublicationStatus/PublicationStatus.scss

```diff
@@ -1,6 +1,3 @@
 .c-status {
-  background: $color-grey-1;
-  text-transform: uppercase;
-  letter-spacing: 0.03rem;
-  font-size: 10px;
+  @apply w-bg-black/50 w-uppercase w-tracking-tight w-text-[10px] w-text-white/85;
```

Comment:
Same as above – I’d prefer we keep the font-size done with CSS, so it stands out more that it really shouldn’t be in pixels, and shouldn’t be so small.